### PR TITLE
Gnome.md stand created

### PR DESCRIPTION
**Pull request by bprakashputta**

### CODE REVIEW

File: content/stands/fosdem/_index.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ new_this_year: |
 
 layout: stand
 ---
-Welcome to the FOSDEM stand!
\ No newline at end of file
+Welcome to the FOSDEM stand!
```
Comment (ovflowd) [inline content/stands/fosdem/_index.md]: Hm, could you revert this file?

File: content/stands/gnome.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+\	our GNOME 40 realease, GUADEC, GNOME.Asia, and Linux App Summit, as well as many more hackfests and social\
+\	events, and selecting our final Community Engagement Challenge winner."
+showcase: "People visiting our stand will have the opportunity to meet GNOME members\
+\	and fellow Community members, discuss new features, address questions, issues,\
+\	or hang out.\r\nNo prior experience using GNOME is needed, and people who are\
+\	looking towards new contribution opportunities are more than welcome."
+themes:
+- Developer environment
+title: GNOME
+website: https://www.gnome.com
```
Comment (ovflowd) [inline content/stands/gnome.md]: Our website is `https://www.gnome.org`

File: content/stands/gnome.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+description: Learn about the latest developments in the GNOME desktop environment! At the GNOME booth, you can check out live demos of GNOME 3.38 and GTK 4, get help using GNOME, and learn more about paid GNOME internship opportunities. During our themed chat hours, you'll be able to engage and talk directly with GNOME developers, designers, Board of Directors members, Foundation staff, and other community members. Ask us your questions or just pop in to say "Hi" and chat! Take a look at our chat schedule - available at our booth - to see what our topics are and when you can join them.
+layout: stand
+logo: stands/gnome/logo.png
```
Comment (ovflowd) [inline content/stands/gnome.md]: Could you move the logo directly to the stands folder? No need to create a new folder called "gnome" the logo could just be called "gnome.png"

**Review by ovflowd — CHANGES_REQUESTED** [review]
Nice changes! 🙌  Just need you to update those stuff that I requested. Thank you! :shipit:

File: content/stands/gnome.md (hunk 2 above)
Comment (bprakashputta) [reply content/stands/gnome.md]: Ooops😅😅

File: content/stands/fosdem/_index.md (hunk 1 above)
Comment (theharrypotter1999) [reply content/stands/fosdem/_index.md:45]: I didn't modify that file, it got modified automatically

File: content/stands/gnome.md (hunk 3 above)
Comment (bprakashputta) [reply content/stands/gnome.md]: Okayy, will name it as gnome-logo.png

**Comment by bprakashputta** [comment]
Okay, will get the changes done. 😅

**Comment by ovflowd** [comment]
> Okay, will get the changes done. 😅

No worries, all good. Take your time 😄

**Comment by bprakashputta** [comment]
Have done the changes requested. you can take a look at them

**Review by ovflowd — APPROVED** [review]

**Comment by bprakashputta** [comment]
First pr done successfully

**Pull request merged**